src/EditorFeatures/Core/Implementation/InlineRename/IEditorInlineRenameService.cs

```diff
@@ -34,6 +35,20 @@ internal enum InlineRenameReplacementKind
         Complexified,
     }
 
+    internal struct InlineRenameFileRenameInfo
+    {
+        public readonly bool KindSupportsFileRename;
+        public readonly bool AllowRename;
```

Comment:
can these be doc'ed.  not sure i understand the distinction and why we need both.   also, of the four combinations, are all supported?  if not, this may benefit from just using an enum.

Reply:
I'm changing to an enum :)